testplan/web_ui/testing/src/Common/utils.js

```diff
@@ -148,8 +148,8 @@ export const flattened = (strArr, ...tagsArr) => {
 export const toPlainObjectIn = (obj, enumerableOnly = false) => {
   const objectify = o => {
     return Object.fromEntries(Object.entries(o)
-      .filter(([_, v]) => _.isPlainObject(v.value))
-      .filter(([_, v]) => !enumerableOnly || v.enumerable)
+      .filter(([__, v]) => _.isPlainObject(v.value))
```

Comment:
`_` is a global variable for `lodash`. Using it as a placeholder for an unused parameter made the placeholder take precedence over the global variable. Hence, this change.